### DISCUSSION TITLE
Проблема с версией 1.35.0-1

### DISCUSSION THREAD

**Discussion by Safronov04** [community/community#84306] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Добрый день.
Помогите разобраться с проблемой. Стоит версия 1.33.1-1 все отлично работает все устройства. Но когда ставлю обновления 1.35.0-1 то zigbee2mqtt начинает постоянно перегружается и после 6 или 8 перезагрузки перестаёт перегружается но странно все работает например пытаюсь включить свет раза 5 или больше он включается и опять уходит в перезагрузки.

**Comment by queenofcorgis** [comment]
Hi @Safronov04 , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be closing any posts in languages other than English for now, including this one.
